src/depends.py

```diff
@@ -363,7 +363,8 @@ def check_curses():
     import subprocess
 
     try:
-        subprocess.check_call('which dialog')
+        subprocess.check_call(       "dialog")
```

Comment:
Why so many spaces? Also did you try '`which dialog`'? @coffeedogs

Reply:
"which dialog"  was a no workie here, but "dialog" alone worked

Reply:
Oh, markdown, I meant
```
"`which dialog`"
```